README.md

```diff
@@ -11,62 +11,65 @@ Advanced keyboard shortcuts for Webflow.io interface
 
 ![gif](https://github.com/maciejsaw/webflow-advanced-keyboard-shortcuts/raw/master/keyboard%20webflow.gif)
 
-1) Open your browser developer tools
+1) Open your browser developer tools (Google Chrome: F12)
```

Comment:
On Windows and Mac the shortcuts are different, I prefer to stick to the Gif "right click" and inspect element for people who don't know what developer tools are